app/models/manager_refresh/inventory_collection_default/infra_manager.rb

```diff
@@ -113,50 +138,125 @@ def ems_clusters(extra_attributes = {})
     def storages(extra_attributes = {})
       attributes = {
         :model_class => ::Storage,
-        :association => :storages,
+        :association => :storages
```

Comment:
It won't be easy since `inventory_collection` do not like my key:
`undefined method `keys' for "spider.eng.lab.tlv.redhat.com:/vol/vol_bodnopoz/data1":String`